src-nextgen/bridges/cc-bridge/container/amd64/x86_32/generic/compcert/v3.1/uberspark-bridge.Dockerfile

```diff
@@ -0,0 +1,110 @@
+FROM ubuntu:18.04
+MAINTAINER David Shepard <djshepard@sei.cmu.edu>
+
+# Also pass in the name of the compiler archive file.
+
+ENV archive_name=compcert-3.1.tgz

```

Comment:
I think that it might be cleaner, if instead of using the `archive_name` variable, you set a `compiler_version` variable. (i.e. instead of `ENV archive_name=compcert-3.1.tgz`, having `compiler_version=compcert-3.1`)

Then you can explicitly add/remove the file format end when required. (e.g., change line 12 to be `RUN wget http://compcert.inria.fr/release/${compiler_version}.tgz`). This might make it where you no longer need the `compiler_script.sh` and can place those few commands (cd, make, make install, etc) into the dockerfile itself.